bot.py

```diff
@@ -338,6 +338,86 @@ async def display_current_list_of_summoners(ctx):
         await ctx.send(embed=create_embed(embed_data))
 
 
+@bot.command(name="teams", help="Display two teams")
+async def display_teams(ctx):
+    """Make and display teams to bot from list of summoners in json"""
+    try:
+        # typing indicator
+        async with ctx.typing():
+            await asyncio.sleep(1)
+
+        # server id
+        server_id = str(ctx.guild.id)
+
+        file_data = ""
+
+        if os.path.getsize(json_path) == 0 or not os.path.exists(json_path):
+            raise Exception
+
+        # get data from the json file and save to file data
+
+        with open(json_path, "r") as file:
+            file_data = json.load(file)
+
+        if not file_data[server_id] or len(file_data[server_id]) != 10:
+            # since when add summoner only accepts up to 10 people
+            raise Exception("NOT ENOUGH PLAYERS")
+
+        teams = make_teams(file_data[server_id])
+
+        # making embed for list of summoners
+        embed_data = EmbedData()
+        embed_data.title = "Teams"
+        embed_data.description = "** **"
+        embed_data.color = discord.Color.green()
+
+        blue_team = teams[0]
+        red_team = teams[1]
+
+        blue_team_output = ""
+        red_team_output = ""
+
+        # using enumerate due to pylint error
+        for count, _ in enumerate(blue_team):
+
+            blue_team_output += "`{0}{1}` {2}\n".format(
+                blue_team[count]["tier_division"][0],
+                TIER_RANK_MAP.get(blue_team[count]["tier_rank_number"]),
+                blue_team[count]["formatted_user_name"],
+            )
+
+            red_team_output += "`{0}{1}` {2}\n".format(
+                red_team[count]["tier_division"][0],
+                TIER_RANK_MAP.get(red_team[count]["tier_rank_number"]),
+                red_team[count]["formatted_user_name"],
+            )
+
+        embed_data.fields = []
+        embed_data.fields.append(
+            {"name": "BLUE", "value": blue_team_output, "inline": True}
+        )
+
+        embed_data.fields.append(
+            {"name": "RED", "value": red_team_output, "inline": True}
+        )
+        await ctx.send(embed=create_embed(embed_data))
+
+    # pylint: disable=broad-except
+    except Exception as e_values:
+        if "NOT ENOUGH" in str(e_values):
```

Comment:
Is there reason why you are not checking for whole Exception message of `"NOT ENOUGH PLAYERS"`?

Reply:
Oh not really, I think it would be neater if I change it to `if str(e_values) == "NOT ENOUGH PLAYERS"`